### ISSUE TITLE
doom apply - variants

### ISSUE THREAD

**Issue by Kasyks** [moo-man/AoS-Soulbound-FoundryVTT#53]
There is no support for apply doom level. Doom can be added to:
mettle -> 1+doom
toughness -> 1+doom
armour ->1+doom
weapon/spell damage -> normal damage+doom

So maybe there is possibility to take Doom from party sheet and apply. 
For me the most easier  way to quick config this  is add it, maybe on stat tab on the right site of skills list, a check boxes which will apply doom and you will have all in one place to it will be much faster to check/change it. 

I assume that UI branch will be a master in the future so it can look like below. If you mark checkbox the chosen "thing" will be recalculated in "doom way".

![obraz](https://user-images.githubusercontent.com/88081888/133231942-d505695c-3dc3-49a1-850e-d886e68308fa.png)

**Comment by Vendare** [comment]
For that to fly the Party Sheet from which to draw the current doom must be configurable since it is possible for a GM to run different groups in the same world. We can't Assume that 1 World = 1 Group.

**Comment by Kasyks** [comment]
Yes you have right, I was think 1world=1group. So current doom on NPC sheet can be set by default to 1 and when the GM will use this  NPC he can just adjust current doom for this NPC. Even when doom will change dynamically during combat GM can change current doom before roll. In both cases it will be much faster then change few things on sheet each time when doom was changed. Even when there will be more then 1 group in the world GM can use same NPC because he will manually change doom for that NPC when needed and enable/disable very quick required doom options for that NPC.
